docs/ADCore/NDArray.rst

```diff
@@ -409,6 +409,41 @@ loading ADBase.template.
     - BAYER_PATTERN
     - $(P)$(R)BayerPattern_RBV
     - mbbi
+  * -
+    -
+    -
+    - **Unique ID and time stamps of the array**
+  * - NDUniqueId
+    - asynInt32
+    - r/o
+    - Unique ID number of array assigned by the EPICS driver.
+    - UNIQUE_ID
+    - $(P)$(R)UniqueId_RBV
+    - longin
+  * - NDTimeStamp
+    - asynFloat64
+    - r/o
+    - Time stamp of array.
+    - TIME_STAMP
+    - $(P)$(R)TimeStamp_RBV
+    - ai
+  * - NDEpicsTSSec
+    - asynInt32
+    - r/o
+    - Seconds past epoch time stamp of array assigned by the EPICS driver. It
+      is based on the EPICS epoch.
+    - EPICS_TS_SEC
+    - $(P)$(R)EpicsTSSec_RBV
+    - longin
+  * - NDEpicsTSNsec
+    - asynInt32
+    - r/o
+    - Nanoseconds time stamp fraction of the EPICS time stamp. It should be
+      composed with $(P)$(R)EpicsTSSec_RBV to get the final time stamp with
+      nanosecond resolution.
+    - EPICS_TS_NSEC
+    - $(P)$(R)EpicsTSNsec_RBV
+    - longin
```

Comment:
You mentioned having to explain the meaning of each parameter in order to differentiate them. Could this documentation simply point to the (ported to rst version of the) areaDetectorTimeStampSupport page? See #537

Reply:
I find it reasonable to deduplicate the explanation about such things. However, I did find the [areaDetectorSupport page](http://htmlpreview.github.io/?https://github.com/areaDetector/ADCore/blob/74716326a0cbdf2907d90d4eeb18f426623edf54/documentation/areaDetectorTimeStampSupport.html) very IOC developer-oriented.

From my perspective, this documentation we are updating has a broader audience. End-users should be able to understand what a PV does without having to dig to much into details. Maybe that's a misconception of my part.

Well, if we are going to reference that page (or an adapted version of it), we still need to restore it. Are you planning to submit a PR for this? Or do you think we should include in the scope of this patch series?